src/vmm/src/arch/aarch64/regs.rs

```diff
@@ -1,80 +1,31 @@
-// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: I'm not 100% sure about the policy, but we tend to set it once when the file is created.